playwright/tests/e2e/features/transaction-detail.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Transaction detail in Cardano explorer
+
+  Scenario: Transaction details page - smart contract info
+    Given the user is in the transaction detail page of a transactions with smart contract info
+    When the user open the contracts section
+    Then the user should see the Contract data
+
+  Scenario: Transaction details page - smart contract info details
+    Given the user is in the transaction detail page of a transactions with smart contract
+    And the user open the contracts section in transaction tab
```

Comment:
Exactly the same thing mentioned before this steps make exactly the same:
**Line 5:** When the user open the contracts section
**Line 10:** And the user open the contracts section in transaction tab
Just reuse the step in Line 5, it doesn't matter if one of the steps use **when** and the other use **and** like in both steps you open the contract section you can simple:
**Line 5:** When the user open the contracts section
**Line 10:** And the user open the contracts section